<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeDisplay</class>
 <widget class="QWidget" name="HomeDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HomeDisplay</string>
  </property>
  <widget class="QWebView" name="newsWebview">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>541</width>
     <height>481</height>
    </rect>
   </property>
   <property name="url">
    <url>
     <string>about:blank</string>
    </url>
   </property>
  </widget>
  <widget class="QLabel" name="dateLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>531</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>45</pointsize>
    </font>
   </property>
   <property name="text">
    <string>saturday september 5th</string>
   </property>
  </widget>
  <widget class="Phonon::VideoPlayer" name="helpVideo" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>781</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QWebView" name="weatherWebview">
   <property name="geometry">
    <rect>
     <x>559</x>
     <y>110</y>
     <width>231</width>
     <height>481</height>
    </rect>
   </property>
   <property name="url">
    <url>
     <string>about:blank</string>
    </url>
   </property>
  </widget>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>541</x>
     <y>1</y>
     <width>251</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>100</pointsize>
    </font>
   </property>
   <property name="text">
    <string>09:44</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="loadProgress">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>781</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
  <customwidget>
   <class>Phonon::VideoPlayer</class>
   <extends>QWidget</extends>
   <header location="global">phonon/videoplayer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
